src/query-builder/transformer/RawSqlResultsToEntityTransformer.ts

```diff
@@ -146,6 +147,36 @@ export class RawSqlResultsToEntityTransformer {
         return hasData;
     }
 
+    protected transformComputedColumns(rawResults: any[], alias: Alias, entity: ObjectLiteral): boolean {
+        let hasData = false;
+
+        this.expressionMap.computedSelects.forEach(computedSelect => {
+            if (computedSelect.aliasName && computedSelect.mapToProperty) {
+                const fields: any[] = computedSelect.mapToProperty.split("_");
+                let entityPart = entity;
+
+                if (fields[0] === alias.name) {
+                    fields.shift();
+                    fields.forEach((field, index) => {
+                        if (index < (fields.length - 1)) {
+                            if (field && !entityPart[field]) {
+                                entityPart[field] = {};
+                            }
+
+                            entityPart = entityPart[field];
```

Comment:
Isn't there a helper that sets deep object values already?

Reply:
I don't know. Could you tell me where is this helper?

Reply:
OrmUtils.mergeDeep